modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/Engine.scala

```diff
@@ -0,0 +1,27 @@
+package edu.gemini.seqexec.engine
+
+import scalaz._
+import Scalaz._
+
+/**
+  * A Map of `Sequence`s.
+  */
+case class Engine[+A](sequences: Map[Sequence.Id, Sequence[A]])
```

Comment:
Is the idea that we'd add more components to engine, like Observing condiitions?